src/components/NewtabPage/BookmarkSearchDialog/BookmarkSearchDialog.tsx

```diff
@@ -95,8 +101,8 @@ export const BookmarkSearchDialog: FC = () => {
             value={searchQuery}
           />
         </form>
-
         {bookmarkChips.length > 0 ? <ChipList chips={bookmarkChips} type={'inline'} /> : undefined}
+        <h1 className={searchDialogTip}>{`Tip: ${tipsArray[Math.floor(Math.random() * 4)]}`}</h1>
```

Comment:
I think `<h1>` is out of place. In terms of semantics it would be better to use `<footer>` here instead.